src/replication.c

```diff
@@ -1974,7 +2006,12 @@ replicate(ISTGT_LU_DISK *spec, ISTGT_LU_CMD_Ptr cmd, uint64_t offset, uint64_t n
 
 		rcomm_cmd->copies_sent++;
 		build_rcmd();
+
+		if (cmd_write)
+			__sync_fetch_and_add(&replica->replica_inflight_write_io_cnt, 1);
```

Comment:
we also need to update `replica_inflight_write_io_cnt` atomically in data_conn.c (in IO response receive path).